src/main/java/com/spring/boot/model/dto/Review.java

```diff
@@ -0,0 +1,23 @@
+package com.spring.boot.model.dto;
+
+import javax.persistence.Column;
+import lombok.Data;
+
+@Data
+public class Review {
+    private Long id;
+    @Column(name = "product_id")
```

Comment:
Looks like an easy solution (not to link to Product table (and to User table for field below). But I think conceptually it is wrong as you have to remember to change both tables when there are new comments or comments is deleted. Though In scope of this task - it might be you have a workable solution.